lib/karafka/serialization/json/deserializer.rb

```diff
@@ -17,8 +17,8 @@ class Deserializer
         #   }
         #   Deserializer.call(params) #=> { 'a' => 1 }
         def call(params)
-          ::MultiJson.load(params.raw_payload)
-        rescue ::MultiJson::ParseError => e
+          ::JSON.parse(params.raw_payload)
```

Comment:
Actually JSON.parse `nil` raises nil compared to JSON.load

Reply:
:facepalm:

Reply:
@nijikon can we add a regression for this? I changed it just because of damn Rubocop :man_shrugging: in the last commit

Reply:
One thing that I can say for sure after all of this, is that I'm by no means a JSON expert @chaadow

Reply:
@mensfeld Neither am I 😂 all good 👌